### ISSUE TITLE
gn3 - remove or hide cookie banner

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#1246]
![screenshot from 2016-02-13 12-00-26](https://cloud.githubusercontent.com/assets/265319/13027295/84b849da-d249-11e5-8244-f03adf098d28.png)

**Comment by fvanderbiest** [comment]
```
<div class="cookie-warning" data-ng-show="true" data-ng-controller="CookieWarningController">...</div>
```

**Comment by landryb** [comment]
Legally, i think you're supposed to show it... even if everyone finds it totally meaningless :)

**Comment by fvanderbiest** [comment]
True, but then, not in gn3 scope.

**Comment by fvanderbiest** [comment]
Fixed with 6c9c1ec
